prisma/schema.prisma

```diff
@@ -114,11 +114,13 @@ model Member {
     messenger         String?
     insta             String?
     photo             String?
+    centre            Centre?                @relation(fields: [centreId], references: [id])
```

Comment:
would this be optional field? Or would there be a default centre for all members that is not deletable? And would it be possible that there be multiple centres for a single member. Just thinking out loud.. maybe we can discuss shortly on next meeting